src/validators/hasMinChecked.js

```diff
@@ -0,0 +1,39 @@
+import logger from '../utils/logger';
+
+/**
+ * @function
+ * @name hasMinChecked
+ * @description A validation method used to check at least n checkboxes of the group are checked.
+ *
+ * @author Timothée Simon-Franza
+ *
+ * @param {array}			checkboxGroupRef	The reference to the checkbox group.
+ * @param {number}			minCheckedAmount	The minimum amount of checkboxes that must be checked.
+ * @param {string}			message				The message to return if the validation fails.
+ *
+ * @returns {string}
+ */
+const hasMinChecked = (minCheckedAmount, message) => (checkboxGroupRef) => {
+	const { isCheckboxGroup, name, rules, ...fields } = checkboxGroupRef;
+	const fieldsAmount = Object.values(fields ?? {}).length;
+
+	if (fieldsAmount === 0) {
+		logger.warn(`tried to check if the ${name} checkbox group has at least ${minCheckedAmount} inputs checked even though it has no checkbox.`);
+
+		return '';
+	}
+
+	if (minCheckedAmount > fieldsAmount) {
+		logger.warn(`tried to check if the ${name} checkbox group has at least ${minCheckedAmount} inputs checked even though it only has ${fieldsAmount} checkboxes.`);
```

Comment:
Handy, is there a reason not to have these checks inside `hasMaxChecked` too ?

Reply:
It didn't occur to me it would be useful since it wouldn't block the form, but it might indeed be useful, I'll implement that